scrapy/resolver.py

```diff
@@ -103,22 +103,22 @@ def from_crawler(cls, crawler, reactor):
     def install_on_reactor(self):
         self.reactor.installNameResolver(self)
 
-    def resolveHostName(
-        self, resolutionReceiver, hostName, portNumber=0, addressTypes=None, transportSemantics="TCP"
+    def resolve_host_name(
+        self, resolution_receiver, host_name, port_number=0, address_types=None, transport_semantics="TCP"
```

Comment:
Besides breaking compatibility in both the method name and the argument names, this no longer implements `IHostnameResolver`.

Reply:
understood. Thank you.